Source/VCGeneration/Counterexample.cs

```diff
@@ -251,6 +240,22 @@ public void PrintModel(TextWriter tw, Counterexample counterexample)
       }
     }
 
+    public void InitializeStates()
+    {
+      if (Model is { ModelHasStatesAlready: false }) {
```

Comment:
Is that the same as `Model?.ModelHasStatesAlready == false`?

Reply:
Yes